apps/deploy.go

```diff
@@ -94,18 +102,20 @@ func (t DeployType) String() string {
 func (d Deploy) Validate() error {
 	var result error
 
-	if d.HTTP == nil &&
-		d.AWSLambda == nil &&
+	if d.AWSLambda == nil &&
+		d.HTTP == nil &&
 		d.Kubeless == nil &&
+		d.OpenFAAS == nil &&
 		d.Plugin == nil {
 		result = multierror.Append(result,
 			utils.NewInvalidError("manifest has no deployment information (http, aws_lambda, open_faas, etc.)"))
 	}
 
 	for _, v := range []validator{
-		d.HTTP,
 		d.AWSLambda,
+		d.HTTP,
```

Comment:
I don't have a strong option on the sorting, but it should be consistent. E.g. `Validate()` is sorted in a different way.

Reply:
fixed to alphabetical throughout the file